src/sentry/static/sentry/app/components/onboardingWizard/todos.jsx

```diff
@@ -145,19 +145,30 @@ class TodoList extends React.Component {
 
   componentWillMount() {
     // Map server task state (who finished what) to TodoList.TASK objects
-    const org = this.props.organization;
+    const {organization} = this.props;
     const tasks = TASKS.map(task => {
-      for (const serverTask of org.onboardingTasks) {
+      for (const serverTask of organization.onboardingTasks) {
         if (serverTask.task === task.task) {
           Object.assign(task, serverTask);
           break;
         }
       }
       return task;
     });
+
+    if (this.hasJavascriptProject) {
+      const uploadSourceMaps = tasks[6];
+      uploadSourceMaps.display = true;
+    }
+
     this.setState({tasks});
   }
 
+  get hasJavascriptProject() {
+    const {organization} = this.props;
+    return organization.projects.some(({platform}) => platform.includes('javascript'));
```

Comment:
Does react-native show up as javascript?

Reply:
No, doesn't look like it. Do we want the source maps task for react-native? Any other SDKs missing (electron, cordova, etc.)? https://docs.sentry.io/_platforms/_index.json

Reply:
I'm pretty sure that react-native and electron can both benefit from source maps.